src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp

```diff
@@ -54,6 +63,39 @@ tnsr::AA<DataType, SpatialDim, Frame> inverse_spacetime_metric(
     const tnsr::II<DataType, SpatialDim, Frame>&
         inverse_spatial_metric) noexcept;
 
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Compute shift from spacetime metric and inverse spatial metric.
+ *
+ * \details Computes
+ * \f{align}
+ *    N^i &= g^{ij} \psi_{jt}
+ * \f}
+ * where \f$ N^i\f$, \f$ g^{ij}\f$, and \f$\psi_{ab}\f$ are the shift, inverse
+ * spatial metric, and spacetime metric.
+ */
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::I<DataType, SpatialDim, Frame> shift(
+    const tnsr::aa<DataType, SpatialDim, Frame>& spacetime_metric,
+    const tnsr::II<DataType, SpatialDim, Frame>&
+        inverse_spatial_metric) noexcept;
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Compute lapse from shift and spacetime metric
+ *
+ * \details Computes
+ * \f{align}
+ *    N &= \sqrt{N^i \psi_{it}-\psi_{tt}}
```

Comment:
Please add a reference to an equation number where this is derived to this comment.

Reply:
I'm not sure where to find an equation number; again I derived this in one line from the expression for g_{00} in terms of lapse and shift.

I could put the entire derivation in the doxygen...

Reply:
That would work!

Reply:
I added a pointer to 2 equations that you can easily derive this expression from.